examples/xlio_ultra_api_ping_pong.c

```diff
@@ -972,9 +832,451 @@ static void process_received_message(xlio_socket_t sock, const char *msg, size_t
     }
 }
 
+/*
+ * ===================================
+ * TX Buffer Pool Management Functions
+ * ===================================
+ *
+ * TX buffer pool is an application logic, but is critical for performance.
```

Comment:
TX buffer pool is an application logic, but is critical for performance. ->
TX buffer pool is application-level logic, but is critical for performance.